### PR TITLE
Added W&B logging

### PR DESCRIPTION
**Pull request by AlekseySh**
- [x] Loggers registry
- [x] Add secrets for Neptune and WandB to GitHub
- [x] Tests for cloud logging (we only test creation in the registry's tests for now, this step requires secrets)
- [x] Add W&B
- [x] Adjust old configs
- [x] Adjust docs 

Also: removed downloading weights in tests for pipelines.

### CODE REVIEW

File: oml/lightning/pipelines/parser.py [hunk 1]
```diff
@@ -60,35 +65,38 @@ def check_is_config_for_ddp(cfg: TCfg) -> bool:
...
-        logger.log_hyperparams(flatten_dict(dict_to_log, sep="|"))
+        logger.log_hyperparams(dict_to_log)
         upload_files_to_neptune_cloud(logger, cfg)
 
+        tags = list(cfg.get("tags", [])) + [cfg.get("postfix", "")] + [cwd]
+        logger.run["sys/tags"].add(tags)
+
+    elif isinstance(logger, WandbLogger):
+        logger.log_hyperparams(dict_to_log)
+        # todo: upload files
```
Comment (AlekseySh) [inline oml/lightning/pipelines/parser.py:92]: todo for @nik-fedorov

File: oml/lightning/callbacks/metric.py [hunk 2]
```diff
@@ -101,11 +102,17 @@ def _log_images(self, pl_module: pl.LightningModule) -> None:
...
+                fig.canvas.draw()
+                fig_img = np.array(fig.canvas.renderer.buffer_rgba())[..., :3]
+                pl_module.logger.log_image(images=[fig_img], key=metric_log_str)
+                plt.close(fig)
             elif isinstance(pl_module.logger, TensorBoardLogger):
                 fig.canvas.draw()
                 data = np.frombuffer(fig.canvas.tostring_rgb(), dtype=np.uint8)
                 data = data.reshape(fig.canvas.get_width_height()[::-1] + (3,))
                 pl_module.logger.experiment.add_image(log_str, np.transpose(data, (2, 0, 1)), pl_module.current_epoch)
+                plt.close(fig)
```
Comment (dapladoc) [inline oml/lightning/callbacks/metric.py:115]: It looks like code duplication. The logic of getting image from figure could be extracted into a function, at used here.

Comment (AlekseySh) [reply oml/lightning/callbacks/metric.py:115]: @nik-fedorov could you take care of this with other todos in the next PR, please?

Comment (nik-fedorov) [reply oml/lightning/callbacks/metric.py:115]: Ok :)

**Pull request merged**